src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionWithNestedActions.cs

```diff
@@ -46,5 +48,11 @@ internal sealed class SuggestedActionWithNestedActions : SuggestedAction
 
         public sealed override Task<IEnumerable<SuggestedActionSet>> GetActionSetsAsync(CancellationToken cancellationToken)
             => Task.FromResult<IEnumerable<SuggestedActionSet>>(NestedActionSets);
+
+        protected override void InnerInvoke(IProgressTracker progressTracker, CancellationToken cancellationToken)
+        {
+            // A code action with nested actions is itself never invokable.  So just do nothing if this ever gets asked.
+            Debug.Fail("Invoke should not be called on a SuggestedActionWithNestedActions");
```

Comment:
my supposition is that somehow the outer wrapping group item got triggered, causing us to eventaully call into GetChangedDocumentAsync below.  That's the only thing i can think of that would be possible.  

The change in this file is to make us resilient to someone calling that.  The change below is to narrow down who is actually throwing if it's something else.

The two pieces of information i'm adding are to try to find out if that's the case.

Reply:
Are we still going to hit `GetChangedDocumentAsync` when running into this scenario you described with this change?

Reply:
> Are we still going to hit GetChangedDocumentAsync when running into this scenario you described with this change?

So one of two thigns is true:

1. Somehow, someone called GetOperations on a 'code action with nested actions' unexpectedly.  If this it eh case, then the override of InnerInvoke here makes us resilient.
2. Someone was calling into a CodeAction that isn't well behaved and is throwing in GetChangedDocument.  If this is the case, then adding the "GetType().FullName" to the exception will tell us which code action is not well behaved.